lib/optional_query_cache/version.rb

```diff
@@ -0,0 +1,11 @@
+require 'active_record'
```

Comment:
This breaks `bundle install` on CI environments, but is required for `OptionalQueryCache` to not have a  superclass mismatch.